examples/vault-cluster-private/main.tf

```diff
@@ -123,7 +123,8 @@ data "template_file" "user_data_consul" {
 # ---------------------------------------------------------------------------------------------------------------------
 
 data "aws_vpc" "default" {
-  default = true
+  default = "${var.vpc_id == "" ? true : false}"
+  id      = "${var.vpc_id}"
```

Comment:
What is this change?

Reply:
see: https://github.com/hashicorp/terraform-aws-consul/blob/master/main.tf#L155

what it does is allow you either specify explicitly an id or default to a randomly created one.

Reply:
changed readme and comments. see below.